plugin-build/src/main/kotlin/io/sentry/android/gradle/extensions/LogcatExtension.kt

```diff
@@ -0,0 +1,21 @@
+package io.sentry.android.gradle.extensions
+
+import io.sentry.android.gradle.instrumentation.logcat.LogcatLevel
+import org.gradle.api.model.ObjectFactory
+import org.gradle.api.provider.Property
+import javax.inject.Inject
+
+open class LogcatExtension @Inject constructor(objects: ObjectFactory) {
+    /**
+     * Enables or disables the Logcat feature.
```

Comment:
Maybe you can add a comment that once enabled, this will replace logcat calls and add breadcrumbs?